cinema/serializers.py

```diff
@@ -30,10 +30,21 @@ class Meta:
         fields = ("id", "name", "rows", "seats_in_row", "capacity")
 
 
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "image"]
+
+
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
```

Comment:
image shouldn't be on MovieSerializer as per task requirements - it must be only on list and retrieve actions